lib/t/matches.ex

```diff
@@ -318,31 +322,42 @@ defmodule T.Matches do
     |> where([match: m], m.user_id_1 == ^user_id or m.user_id_2 == ^user_id)
     |> where([match: m], m.id not in subquery(archived_match_ids_q(user_id)))
     |> order_by(desc: :inserted_at)
-    |> join(:left, [match: m], t in Timeslot, as: :t, on: m.id == t.match_id)
-    |> join(:left, [match: m], c in MatchContact, as: :c, on: m.id == c.match_id)
-    |> select([match: m, undying_event: e, t: t, c: c], {m, t, c, e.timestamp})
+    |> join(:left, [m], t in assoc(m, :timeslot), as: :t)
+    |> join(:left, [m], c in assoc(m, :contact), as: :c)
+    |> join(:left, [m], v in assoc(m, :voicemail), as: :v, on: v.caller_id != ^user_id)
```

Comment:
client needs to receive `voicemail` even he was the "caller" - we do not use the messages themselves right now, but just the fact that he was the last voicemail sender and haven't received a reply yet

so I would suggest using `picker` approach similar to how we do it for `timeslot` and `contact`

Reply:
@akhrail1996 ok if I include `caller` in the payload, not `picker`?

Reply:
@ruslandoga sure

Reply:
@akhrail1996 https://github.com/getsince/test3/pull/493